sdks/python/setup.py

```diff
@@ -62,6 +62,7 @@ def get_version():
     'oauth2client>=2.0.1',
     'protorpc>=0.9.1',
     'python-gflags>=2.0',
+    'python-snappy>=0.5',
```

Comment:
Cham, how would we go about specifying 'libsnappy-dev' which is a dependency for this?